.github/workflows/e2e-nvidia-a10g-x4.yml

```diff
@@ -164,16 +164,29 @@ jobs:
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Post job results to Slack
-        if: ${{ !cancelled() && steps.check_pr.outputs.is_pr == 'false' }}
-        id: slack
+      - name: Post job results to Slack if the workflow failed
+        if: failure() && steps.check_pr.outputs.is_pr == 'false'
+        id: slack-failure
```

Comment:
Would slack-report-failure be more appropriate?

Reply:
SGTM